src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -53,6 +53,9 @@
     -webkit-font-smoothing: subpixel-antialiased;
     -moz-osx-font-smoothing: auto;
 }
+div.sf-toolbar .sf-toolbarreset a {
```

Comment:
Shouldn't you rather update the existing `.sf-toolbar-block .sf-toolbar-info-piece a` selector?

Reply:
When doing so, it adds an underline on text icons:

![screenshot-2018-6-29 welcome](https://user-images.githubusercontent.com/1588144/42102729-1742499e-7bc7-11e8-8067-197fab58057c.png)

I think this is not the "same" link I want to target

Reply:
Ok I get your point, thank you.

It works on 2.8